registry/registry.py

```diff
@@ -338,12 +350,12 @@ def to_json(self):
         """
         Writes out the registry as a JSON object, perhaps to be served by API
         server.
-        I think we should *not* send back the whole model or env, because they
-        will contain all of the other agents. Also, groups should just send
-        back the group members names.
-        Only individual agents should be fully represented in the returned
-        JSON.
+        For now, we will just do exec_keys and model names.
         """
+        ret_json = {}
+        for key in self.registries:
```

Comment:
In case of multi threading, it could happen that a particular registry is on disk but not in this thread.

Reply:
Let's get this version out in production and then add a load from disk as an enhancement.